server/src/analyzer/expressionEvaluator.ts

```diff
@@ -849,6 +849,28 @@ export class ExpressionEvaluator {
         let validatedTypes = false;
         let returnType: Type | undefined;
 
+        if (TypeUtils.isDataClass(type)) {
+          let constructorMethodType: FunctionType = new FunctionType(FunctionTypeFlags.InstanceMethod);
```

Comment:
Nit: No need to specify type here because typescript can infer it correctly. Linter will complain about this because it's an unnecessary type annotation.